baseplate/lib/metrics.py

```diff
@@ -135,14 +135,14 @@ def __init__(self, transport: Transport, namespace: str):
         self.transport = transport
         self.namespace = namespace.encode("ascii")
 
-    def timer(self, name: str) -> "Timer":
+    def timer(self, name: str, sample_rate: float = 1.0) -> "Timer":
```

Comment:
this makes timers have a pretty different interface from counters. would it make sense to put the sample rate on the `start()` and `send()` methods of a timer? feels kinda weird too.

Reply:
Yeah I thought about this, I'd be happy to move it to `start`. I wasn't sure if the same timer could get restarted or how that might interact.